### PR TITLE
tools: update config for reparo

### PR DESCRIPTION
**Pull request by WangXiangUSTC**
### What is changed, added or deleted? 
add two config

### Which version does your change affect? 

update dev at the first, after get two LGTM I will update v2.1 and v3.0

### CODE REVIEW

File: dev/reference/tools/tidb-binlog/reparo.md [hunk 1]
```diff
@@ -18,6 +18,8 @@ Usage of Reparo:
 -L string
     日志输出信息等级设置：debug, info, warn, error, fatal (默认值：info)。
 -V 打印版本信息。
+-c int
+    同步下游的并发数，该值设置越高同步的吞吐性能越好 (默认 16)
```
Comment (CaitinChen) [inline dev/reference/tools/tidb-binlog/reparo.md]:
```suggestion
    同步下游的并发数，该值设置越高同步的吞吐性能越好（默认 16）。
```

File: dev/reference/tools/tidb-binlog/reparo.md [hunk 2]
```diff
@@ -18,6 +18,8 @@ Usage of Reparo:
 -L string
     日志输出信息等级设置：debug, info, warn, error, fatal (默认值：info)。
```
Comment (CaitinChen) [inline dev/reference/tools/tidb-binlog/reparo.md]:
```suggestion
    日志输出信息等级设置：debug, info, warn, error, fatal（默认值：info）。
```

File: dev/reference/tools/tidb-binlog/reparo.md [hunk 3]
```diff
@@ -34,6 +36,8 @@ Usage of Reparo:
     用于指定结束恢复的时间点，格式同上。如果不设置该参数则恢复到最后一个 binlog 文件。
 -safe-mode bool
     指定是否开启安全模式，开启后可支持反复同步。
+-txn-batch int
+    输出到下游数据库一个事务的 SQL 语句数量 (默认 20)
```
Comment (CaitinChen) [inline dev/reference/tools/tidb-binlog/reparo.md]:
```suggestion
    输出到下游数据库一个事务的 SQL 语句数量（默认 20）。
```

File: dev/reference/tools/tidb-binlog/reparo.md [hunk 4]
```diff
@@ -61,6 +65,12 @@ log-level = "info"
 # 下游服务类型。 取值为 print, mysql（默认值：print）。当值为 print 时，只做解析打印到标准输出，不执行 SQL；如果为 mysql，则需要在 [dest-db] 中配置 host、port、user、password 等信息。
 dest-type = "mysql"
 
+# 输出到下游数据库一个事务的 SQL 语句数量 (默认 20)
```
Comment (CaitinChen) [inline dev/reference/tools/tidb-binlog/reparo.md]:
```suggestion
# 输出到下游数据库一个事务的 SQL 语句数量（默认 20）。
```

File: dev/reference/tools/tidb-binlog/reparo.md [hunk 5]
```diff
@@ -61,6 +65,12 @@ log-level = "info"
...
+txn-batch = 20
+
+# 同步下游的并发数，该值设置越高同步的吞吐性能越好 (默认 16)
```
Comment (CaitinChen) [inline dev/reference/tools/tidb-binlog/reparo.md]:
```suggestion
# 同步下游的并发数，该值设置越高同步的吞吐性能越好（默认 16）。
```

**Comment by WangXiangUSTC** [comment]
@CaitinChen thanks, PTAL again

**Pull request merged**